src/main/java/org/mitre/synthea/world/agents/Person.java

```diff
@@ -127,6 +130,7 @@
   public Person(long seed) {
     this.seed = seed; // keep track of seed so it can be exported later
     random = new Random(seed);
+    mathRandom = new JDKRandomGenerator((int) seed);
```

Comment:
I would rather you just change `random` to be an instance of `JDKRandomGenerator`, since it just extends the random class anyway and adds an extra interface definition because apache commons math3 has to be special.

Reply:
That makes sense and I can do that.